maestro/demos/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+# Contributing to Bee Hive Demos
+
+Thank you for your interest in contributing to **Bee Hive**! This document outlines the structure and workflow for setting up and testing demos using **Maestro**.
+
+---
+
+## Project Structure
+
+```markdown
+/maestro
+│── /src                    # Source code for core functionalities
+│   │── agent.py            # Base class for defining agents
+│   │── bee_agent.py        # Implementation of BeeAgent
+│   │── create_agents.py    # Creates agents from YAML definitions
+│   │── run_workflow.py     # Executes workflows sequentially
+│   │── step.py             # Step execution logic for workflows
```

Comment:
Why do we need to include this in contributing for demos? This should not be code demos use directly, ideally they never do. Only time I could see if if they want to extend the base agent class